src/test/java/org/mastodon/mamut/collaboration/utils/ReloadFromDiskUtilsTest.java

```diff
@@ -0,0 +1,52 @@
+package org.mastodon.mamut.collaboration.utils;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+
+import org.junit.Test;
+import org.mastodon.mamut.ProjectModel;
+import org.mastodon.mamut.collaboration.TestResources;
+import org.mastodon.mamut.io.ProjectLoader;
+import org.mastodon.mamut.io.project.MamutProjectIO;
```

Comment:
```suggestion
```
This import is obsolete

Reply:
Thank you for spotting this. I removed the line